src/main/java/seedu/address/logic/Messages.java

```diff
@@ -18,6 +18,7 @@ public class Messages {
     public static final String MESSAGE_PERSONS_LISTED_OVERVIEW = "%1$d persons listed!";
     public static final String MESSAGE_DUPLICATE_FIELDS =
                 "Multiple values specified for the following single-valued field(s): ";
+    public static final String MESSAGE_INVALID_PERSON = "No person found with this keyword";
```

Comment:
Hm, in Messages there is already a constant defined that says almost the same thing. If it is not showing for you, you might need to sync merge thing again as this already in the latest main branch

![image](https://github.com/AY2324S2-CS2103T-W12-4/tp/assets/104015990/7d3d2a4b-e789-4ad9-a8a8-6885150536a9)